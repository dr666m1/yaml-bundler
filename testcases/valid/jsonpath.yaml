foo: !include
  filepath: ./sub/jsonpath.yaml
  jsonpath: $.foo.bar
qux: !include
  filepath: ./sub/jsonpath.yaml
  jsonpath: $.qux
grault: !include
  filepath: ./sub/jsonpath.yaml
  jsonpath: $.qux[0]
garply: !include
  filepath: ./sub/jsonpath.yaml
  jsonpath: $.qux[*]
