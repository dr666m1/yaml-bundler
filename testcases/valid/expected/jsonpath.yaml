foo: baz
qux:
  - quux
  - corge
grault: quux
garply:
  - quux
  - corge
